components/authz-service/server/v2/policy_test.go

```diff
@@ -1099,13 +1017,13 @@ func TestUpdatePolicy(t *testing.T) {
 				Effect:    api_v2.Statement_ALLOW,
 				Resources: []string{"compliance:profiles"},
 				Actions:   []string{"compliance:profiles:upload"},
-				Projects:  []string{constants_v2.AllProjectsExternalID}, // explicit "all"
+				Projects:  []string{constants_v2.AllProjectsExternalID},
 			}
 			statement1 := api_v2.Statement{
 				Effect:    api_v2.Statement_ALLOW,
 				Resources: []string{"cfgmgmt:delete", "cfgmgmt:list"},
 				Actions:   []string{"cfgmgmt:nodes:*"},
-				Projects:  []string{}, // implicit "all"
+				Projects:  []string{constants_v2.UnassignedProjectID}, // implicit "all"
```

Comment:
Need to remove the comment
```suggestion
				Projects:  []string{constants_v2.UnassignedProjectID},
```